### ISSUE TITLE
Adding a small file fails. The error does not help much. 

### ISSUE THREAD

**Issue by felixniemeyer** [textileio/powergate#793]
After setting up local net and the CLI-tool as described in the docs, adding ("applying config to") a small file as described in the docs fails.  

Reproduce: 

setup localnet and pow cli like described in the docs. Create a small file, create a user, stage the file and apply the config like this: 
```bash
echo "test" > testfile
response=`pow admin user create`
token=`jq -r '.[] | .token' <<< $response`
export POW_TOKEN=$token
response=`pow data stage testfile`
cid=`jq -r '.cid' <<< $response`
pow config apply --watch $cid
```

And it will fail with this job status: 
```
JOB_STATUS_FAILED executing cold-storage config:
 making deal configs:
 getting miners from minerselector:
 getting miners from reputation module:
 not enough miners satisfy the miner selector constraints 
```

When altering the code a bit to return an error as soon as one miner does not satisfy the constraints you learn that the piece size is too small: 
```
JOB_STATUS_FAILED executing cold-storage config:
 making deal configs:
 getting miners from minerselector:
 getting miners from reputation module:
 skipping cached miner, not replying to miner proposal query:
 miner doesn't satisfy piece size constraints:
 256<128<536870912       
```

When trying to add a bigger file, everything works. 

This may leave new users puzzled for a while.

Maybe adding a hint about the file size to the docs would be helpful enough. 
Or maybe doing something with this error in `reptop.go` [line 97](https://github.com/textileio/powergate/blob/a2048ee8d580d16b17698358d0ade3f1285150ee/ffs/minerselector/reptop/reptop.go#L98)
```go
if err != nil {
	continue // Cached miner isn't replying to query-ask now, skip.
}
```

The current handling might also be disadvantageous in other situations: if you have chosen impossible miner requirements, it would be good to know, what requirements exactly is hard to satisfy.

I have a solution in mind and will follow up with a pull request.

**Comment by jsign** [comment]
Hey @felixniemeyer, thanks for the detailed explanation.

This is already explained in the docs [here](https://docs.textile.io/powergate/localnet/#localnet-features):
> Not using --bigsectors will limit you to storing files of around 700 bytes and deals will complete in 30-60 seconds. Using --bigsectors will allow you to store files anywhere from 1Mb to 400Mb, but deals will complete in 3-4 minutes. Be sure to choose the value that makes sense for your development scenario.

We also make sone reference in Powergate's [README](https://github.com/textileio/powergate#localnet-mode), where we explain a way to use 2KiB sectors which will allow to store small files (actually, the size we use for Powergate CI):
> Note: Running BIGSECTORS=false make localnet will create the Lotus devent using 2Kib sectors. This may be more appropriate for certain development or testing scenarios.

Despite that, I don't like that you lost some time not finding this information. Do you feel we can refactor a bit where this information is explained, or improve it in some way?

**Comment by felixniemeyer** [comment]
Hey jsign, 

oh you're right! I even remember reading this paragraph. 
Maybe add a hint here as well: https://docs.textile.io/powergate/storage/#make-data-available

Also, seeing the exact reason why the localnet miner was not satisfying the selector constraints pointed me in the right direction. So I think showing some of the miner rejection reasons would be good, if no suitable miner was found. 

That's implemented in this pull request: https://github.com/textileio/powergate/pull/794

**Comment by jsign** [comment]
Closed by #794
